win32ss/user/ntuser/menu.c

```diff
@@ -2939,7 +2939,7 @@ static BOOL MENU_MoveRect(UINT flags, INT* x, INT* y, INT width, INT height, con
 static BOOL FASTCALL MENU_ShowPopup(PWND pwndOwner, PMENU menu, UINT id, UINT flags,
                               INT x, INT y, const RECT* pExclude)
 {
-    INT width, height;
+    INT width, height, AdjHgt;
```

Comment:
instances of variables should begin not with capital letters 1/4
```suggestion
    INT width, height, adjHgt;
```

Reply:
Fixed. Thanks.